site/content/media-kit.md

```diff
@@ -7,28 +7,37 @@ description: Download our assets and discover our style guide.
 
 ## Logos
 
-### Version 1
+### Version 1 - Rectangular
 
-![Logo - V1](https://i.imgur.com/ATxChJD.png)
+![Logo - ](https://i.imgur.com/VCqpxcT.png)
 
-### Version 2
+### Version 2 - Rectangular with no URL
 
-![Logo - V2](https://i.imgur.com/mS3GR9l.png)
+![Logo - V2](https://i.imgur.com/qny5XYN.png)
 
-### Version 3
+### Version 3 - Squared
+
+![Logo - V3](https://i.imgur.com/eFTD651.png)
+
+### Version 4 - Squared No URL
+
+![Logo - V4](https://i.imgur.com/RRVpP6V.png)
 
-![Logo - V3](https://i.imgur.com/mLO2a2t.png)
 
 ## Banners
 
 ### Banner 1
 
-![Banner 1](https://i.imgur.com/A43r74s.png)
+![Banner 1](https://i.imgur.com/QZAbJUJ.png)
```

Comment:
The banner still looks a little off - mind if I update it so it's vertically and horizontally centered?

![Banner](https://i.imgur.com/QZAbJUJ.png)

Reply:
Weird, I thought I was using the guides correctly. I'll send you a PSD

Reply:
We can replace this with https://i.imgur.com/ONfQqEV.png

![](https://i.imgur.com/ONfQqEV.png)